Objects/Objects/BuiltElements/Pipe.cs

```diff
@@ -51,11 +52,12 @@ public class RevitPipe : Pipe
     public Base parameters { get; set; }
     public string elementId { get; set; }
     public Level level { get; set; }
+    public Connector[] connectors { get; set; }
 
     public RevitPipe() { }
 
     [SchemaInfo("RevitPipe", "Creates a Revit pipe", "Revit", "MEP")]
-    public RevitPipe(string family, string type, [SchemaMainParam] ICurve baseCurve, double diameter, Level level,  string systemName = "", string systemType = "", List<Parameter> parameters = null)
+    public RevitPipe(string family, string type, [SchemaMainParam] ICurve baseCurve, double diameter, Level level, string systemName = "", string systemType = "", List<Parameter> parameters = null, Connector[] connectors = null)
```

Comment:
A new constructor needs to be added with the `connectors` argument to not break backwards compatibility with existing GH components